languages/ruby/outline.scm

```diff
@@ -5,9 +5,21 @@
 ((identifier) @context
   (#match? @context "^(private|protected|public)$")) @item
 
-(method
-    "def" @context
-    name: (_) @name) @item
+(body_statement
+  (call
+      method: (identifier) @context
```

Comment:
Not selecting a specific identifier here since it should allow for arbitrary modifiers, which can be defined as class methods taking a symbol as a positional argument.